augmentation.py

```diff
@@ -9,6 +9,8 @@
 
 --augment_level 2-8: will generate flipped / rotated images
 
+Don't move this script or it can't import from helper.
```

Comment:
Do we need this comment? I think it's very obvious that you shouldn't move the file which is referenced from others.